ecosystem/sep-0006.md

```diff
@@ -1123,6 +1180,9 @@ Name | Type | Description
 `refunds` | object | (optional) An object describing any on or off-chain refund associated with this transaction. The schema for this object is defined in the [Refunds Object Schema](#refunds-object-schema) section below.
 `required_info_message` | string | (optional) A human-readable message indicating any errors that require updated information from the user.
 `required_info_updates` | object | (optional) A set of fields that require update from the user described in the same format as [/info](#info).  This field is only relevant when `status` is `pending_transaction_info_update`.
+`required_customer_info_message` | string | (optional) A human-readable message indicating any errors that require updated information from the user.
```

Comment:
I think this field will be used in the case where businesses haven't attempted to facilitate the transaction yet, so I think we should rephrase the description to remove mention of errors.